branchLayout/impl/src/main/java/com/virtuslab/branchlayout/impl/readwrite/BranchLayoutWriter.java

```diff
@@ -0,0 +1,59 @@
+package com.virtuslab.branchlayout.impl.readwrite;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+import io.vavr.collection.List;
+import lombok.CustomLog;
+import lombok.RequiredArgsConstructor;
+import lombok.val;
+import org.checkerframework.checker.index.qual.NonNegative;
+
+import com.virtuslab.branchlayout.api.BranchLayout;
+import com.virtuslab.branchlayout.api.BranchLayoutEntry;
+import com.virtuslab.branchlayout.api.readwrite.IBranchLayoutWriter;
+import com.virtuslab.branchlayout.api.readwrite.IndentSpec;
+import com.virtuslab.qual.guieffect.IgnoreUIThreadUnsafeCalls;
+
+@CustomLog
+@RequiredArgsConstructor
```

Comment:
Nit: there are no required args, right? so the `@RequiredArgsConstructor` will just be the default zero-param c'tor (which is generated by default anyway)?